recipes-egt/apps/egt-benchmark/0001-configure-Do-not-add-C-compiler-flags-to-CPPFLAGS.patch

```diff
@@ -0,0 +1,30 @@
+From dc1a6e863f096c89e80cfb536fb7a65072e06efc Mon Sep 17 00:00:00 2001
+From: Khem Raj <raj.khem@gmail.com>
+Date: Wed, 6 May 2020 22:38:16 -0700
+Subject: [PATCH] configure: Do not add C++ compiler flags to CPPFLAGS
+
+CPPFLAGS is for c pre-processor and thusly added to CFLAGS as as well as
+CXXFLAGS, -std=c++14 is only valid for C++ compiler, therefore do not
+add it to CPPFLAGS
+
```

Comment:
Thanks Raj! Could you also please send this patch to https://github.com/linux4sam/egt-thermostat ?

Reply:
I guess you meant egt-benchmark then yes its already sent see https://github.com/linux4sam/egt-benchmark/pull/1

Reply:
Yes, egt-benchmark. Thanks!